subprojects/docs/src/docs/release/notes.md

```diff
@@ -387,13 +387,14 @@ In Gradle 7.0 we moved the following classes or methods out of incubation phase.
         - org.gradle.tooling.events.test.Destination
         - org.gradle.tooling.events.test.TestOutputDescriptor
         - org.gradle.tooling.events.test.TestOutputEvent
+  - Debugging
```

Comment:
Is the indentation wrong now? It looks like it is one less level.

Reply:
Indeed, it misses a couple spaces. One more good reason to use tabs :)

Reply:
Hmm..., wouldn't it then miss a couple of tabs?